include/scale/scale_decoder_stream.hpp

```diff
@@ -293,103 +248,88 @@ namespace scale {
      */
     ScaleDecoderStream &operator>>(BitVec &v);
 
+    ScaleDecoderStream &operator>>(DynamicSpan auto &collection) {
```

Comment:
There was no such function before.
Is it needed?

Does it set pointer to decoder buffer which may expire, or assume span points to some writable buffer?